src/pages/ProductListing/Filters/Filters.jsx

```diff
@@ -0,0 +1,143 @@
+import "./Filters.css";
+export const Filters = () => {
+	return (
+		<aside className="aside aside--products">
+			<form action="post" className="filter">
+				<header className="filter__header">
+					<h3 className="h5 filter__head">Filters</h3>
+					<input type="reset" value="Clear All" className="filter--reset" />
+				</header>
+
+				<section className="filter__section">
+					<h3 className="h5 filter__head">Price</h3>
+
+					<ul className="range__head">
+						<li className="range__amount">500</li>
+						<li className="range__amount">1500</li>
+						<li className="range__amount">2500</li>
+					</ul>
+
+					<input
+						type="range"
+						min="500"
+						max="2500"
+						defaultValue="1500"
+						className="range__sliders"
+					/>
+				</section>
+				<section className="filter__section">
+					<h3 className="h5 filter__head">Category</h3>
+
+					<ul className="filter__items">
+						<li className="filter__list">
+							<input
+								type="checkbox"
+								id="category1"
+								className="filter__checkbox"
+							/>
+							<label htmlFor="category1" className="filter__item_label">
+								Cricket Bats
+							</label>
+						</li>
```

Comment:
You can create `input` component here and reuse it.

Reply:
Nice suggestion. will consider doing that.